include/drivers/espi_emul.h

```diff
@@ -152,6 +165,18 @@ int emul_espi_host_send_vw(const struct device *espi_dev,
  */
 int emul_espi_host_port80_write(const struct device *espi_dev, uint32_t data);
 
+#ifdef CONFIG_ESPI_PERIPHERAL_ACPI_SHM_REGION
+/**
+ * Get the host device's ACPI shared memory start address. The size of the region is
+ * CONFIG_EMUL_ESPI_HOST_ACPI_SHM_REGION_SIZE.
+ *
+ * @param espi_dev eSPI host emulator.
```

Comment:
The other similar functions(`emul_espi_host_send_vw` and `emul_espi_host_port80_write`) take a pointer to the eSPI emulation controller not host. I think we should keep this consistent.

Reply:
Done, sorry for the confusion. I thought we wanted to be able to get the address via the emulator directly.

Reply:
Please, change the comment to eSPI controller emulator.